warehouse/query-core/src/main/java/datawave/query/iterator/builder/IndexRegexIteratorBuilder.java

```diff
@@ -19,7 +19,7 @@
  *
  */
 public class IndexRegexIteratorBuilder extends IvaratorBuilder implements IteratorBuilder {
-    private static Logger log = Logger.getLogger(IndexRegexIteratorBuilder.class);
+    private static final Logger log = Logger.getLogger(IndexRegexIteratorBuilder.class);
```

Comment:
there's an MR out to refactor the Logger references to the slf4j LoggerFactory instead, so it may be worth updating this to be in line with what we expect the loggers to be built as going forward